### DISCUSSION TITLE
Error running plot in spatial coordinates

### DISCUSSION THREAD

**Discussion by ChristinaSteyn** [BayraktarLab/cell2location#246] [Q&A]
Hi there, 

I was also hoping to get assistance with this code:

```
with mpl.rc_context({'axes.facecolor':  'black',
                     'figure.figsize': [4.5, 5]}):
    sc.pl.spatial(adata_vis, color=['region_cluster'],
                  size=1.3, img_key='hires', alpha=0.5, save = '_region_cluster_spatial_coords.pdf')
```

which is giving this error:

ValueError: Found multiple possible libraries in `.uns['spatial']. Please specify. Options are:
        ['run_spaceranger_count_DH1', 'run_spaceranger_count_DH1a', 'run_spaceranger_count_DH2', 'run_spaceranger_count_DH2a', 'run_spaceranger_count_DH3', 'run_spaceranger_count_DH3a', 'run_spaceranger_count_DH4', 'run_spaceranger_count_DH4a']

It seems fairly straightforward but I am a novice in python and not sure how I should go about specifying one of these. 

Thank you in advance!

**Comment by vitkl** [comment]
Please check scanpy documentation and spatial tutorial https://scanpy-tutorials.readthedocs.io/en/latest/spatial/basic-analysis.html
